cloud/src/rate-limiter/rate_limiter.cpp

```diff
@@ -93,6 +154,35 @@ bool RpcRateLimiter::get_qps_token(const std::string& instance_id,
     return qps_token->get_token(get_bvar_qps);
 }
 
+void RpcRateLimiter::set_max_qps_limit(int64_t max_qps_limit) {
+    std::lock_guard<bthread::Mutex> l(mutex_);
+    max_qps_limit_ = max_qps_limit;
+    auto filter = [this](const auto& kv) {
+        return !instance_with_specific_limit_.contains(kv.first);
+    };
+    for (auto& [k, v] : qps_limiter_ | std::views::filter(std::move(filter))) {
+        v->set_max_qps_limit(max_qps_limit);
+    }
+}
+
+bool RpcRateLimiter::set_max_qps_limit(int64_t max_qps_limit, const std::string& instance_id) {
```

Comment:
"limit" already implies "max", "max" in the name seems redundant

Reply:
just inheritate the original naming, will be mod